_opensearch/query-dsl/query-filter-context.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Query and filter context
+parent: Query DSL
+nav_order: 5
+---
+
+# Query and filter context
+
+Queries consist of query clauses, which can be run in a [_filter context_](#filter-context) or [_query context_](#query-context). A query clause in a filter context asks the question "_Does_ the document match the query clause?" and returns matching documents. A query clause in a query context asks the question "_How well_ does the document match the query clause?" and returns matching documents, providing the relevance of each document in the [_relevance score_](#relevance-score).
+
+## Relevance score
+
+A _relevance score_ measures how well a document matches a query. It is a positive floating-point number that OpenSearch records in the `_score` metadata field for each document. A higher score corresponds to a more relevant document. While different query types calculate relevance scores differently, all query types take into account whether a query clause it is run in a filter or query context. 
+
+Use query clauses that should affect the relevance score in a query context, and all other query clauses in a filter context.
```

Comment:
Use query clauses that have an effect on the relevance score in a query context, and use all other query clauses in a filter context.

Does this preserve the meaning?

Reply:
I don't think it's quite the same. Whether or not a query clause affects the relevance score is not intrinsic to a query clause. If you want it to affect the relevance score, you should put it in a query context, and if you don't, then you put it in a filter context.

Reply:
Do we mean something like "Use query clauses that you want to affect the relevance score in a query context, and use all other query clauses in a filter context"?